pkg/controller/machineset/machineset_controller.go

```diff
@@ -150,12 +150,7 @@ func (r *ReconcileMachineSet) Reconcile(request reconcile.Request) (reconcile.Re
 }
 
 func (r *ReconcileMachineSet) reconcile(ctx context.Context, machineSet *clusterv1.MachineSet) (reconcile.Result, error) {
-	klog.V(4).Infof("Reconcile machineset %v", machineSet.Name)
-	allMachines := &clusterv1.MachineList{}
-
-	if err := r.Client.List(context.Background(), allMachines, client.InNamespace(machineSet.Namespace)); err != nil {
-		return reconcile.Result{}, errors.Wrap(err, "failed to list machines")
-	}
+	klog.V(4).Infof("Reconcile MachineSet %q in namespace %q", machineSet.Name, machineSet.Namespace)
```

Comment:
Just a nit, but why not use `<namespace>/<name>` here?

Reply:
The pattern is used elsewhere as well, I can follow-up to clean it up and use the one you suggested instead